adapter/spvadapter/spvadapter.cpp

```diff
@@ -405,11 +404,18 @@ void SpvDidAdapter_CreateIdTransactionEx(SpvDidAdapter *adapter,
         tx = adapter->idWallet->SignTransaction(tx, password);
         tx = adapter->idWallet->PublishTransaction(tx);
         std::string txid = tx["TxHash"];
-        adapter->callback->RegisterTransactionCallback(txid,
-                confirms, txCallback, context);
+        /*if (txid.empty())
```

Comment:
If it's OK to create ID transaction, do we need to call SPvTransactionCallback to tell upperlayer about this success ?